packages/react/src/hooks/useMedia.tsx

```diff
@@ -0,0 +1,44 @@
+import { useLayoutEffect, useState } from 'react'
+
+interface UseMediaType {
+  mobile: boolean
+  tablet: boolean
+  desktop: boolean
+}
+
+const useMedia = (): UseMediaType => {
+  const [isTablet, setIsTablet] = useState(
+    typeof window !== 'undefined' &&
```

Comment:
```suggestion
const isClient = typeof window !== 'undefined' 
   isClient &&
```
전체적으로 client일 때를 확인하고 있어서 변수처리 해야할 것 같아요!